example/settings.py

```diff
@@ -101,7 +101,8 @@ def load_conf(cfg):
     },
     'crate': {
         'ENGINE': 'django_crate.backend',
-        'SERVERS': ['localhost:4200',],
```

Comment:
there are ways to do connection pooling with psycopg. however, to achieve HA the recommended setup is to run both the application and a cratedb node on the same host, where the application is talking to localhost and the database nodes are talkin to each over.
see https://crate.io/docs/reference/architecture/shared_nothing.html#application-use-case